### ISSUE TITLE
[A11Y] [Web] Share Icons on Component Page not labeled

### ISSUE THREAD

**Issue by shengoogle** [ampproject/ampstart#417]
ENVIRONMENT:
Build number: 56.0.2924.87 beta (64-bit)
Form factor: Chromebook
Browser: Chrome

PRE-REQUISITE
chromevox is ON

STEPS TO REPRODUCE:
1. Open https://ampstart.com
2. Navigate to Components Tab
3. Using chromevox, navigate to Share Icons
4. Cycle through Share Icons

ACTUAL RESULTS:
Share Icons read as link

EXPECTED RESULTS:
Icon should read properly ie bird icon for twitter etc.

ATTACHMENT:

**Comment by torch2424** [comment]
In my a11y bugs branch

[Commit Fix](https://github.com/torch2424/ampstart/commit/47b798caab36b9c893752f537285e1ea1b22f2e9)
